### PR TITLE
osm-deployment: parametrize osm log level in values.yaml

### PR DESCRIPTION
**Pull request by eduser25**
Currently it's hardcoded in deployment.

This gives at least some short term flexibility till we add
configMap update, which has some ongoing discussion as
how to tackle.

- Install                [X]
- Control Plane          [X]

- Does this change contain code from or inspired by another project? If so, did you notify the maintainers and provide attribution?
No

### CODE REVIEW

File: charts/osm/values.yaml [hunk 1]
```diff
@@ -43,6 +43,7 @@ OpenServiceMesh:
   meshName: osm
   useHTTPSIngress: false
   envoyLogLevel: error
+  osmLogLevel: trace
```
Comment (shashankram) [inline charts/osm/values.yaml:46]: should we rename this to `controllerLogLevel`?

**Review by shashankram — APPROVED** [review]

File: charts/osm/values.yaml (hunk 1 above)
Comment (ksubrmnn) [reply charts/osm/values.yaml:46]: Agreed, otherwise it may be confused with envoy log level

**Review by ksubrmnn — APPROVED** [review]

File: charts/osm/values.yaml (hunk 1 above)
Comment (eduser25) [reply charts/osm/values.yaml:46]: ... I guess? we didn't call `envoyLogLevel` `sidecarProxyLogLevel`, but if you feel this is clearer, I'll change it gladly

Comment (eduser25) [reply charts/osm/values.yaml]: since we use it as `OpenServiceMesh.controllerLogLevel`, seems more than ok. changing it.

**Review by snehachhabria — APPROVED** [review]

**Review by shashankram — APPROVED** [review]

**Pull request merged**